src/styles/blocks/stars--2.css

```diff
@@ -0,0 +1,44 @@
+.stars--2 {
+  display: flex;
+  margin-top: 4px;
+}
+
+.stars--2 .stars__star:nth-child(1) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
```

Comment:
You have repeated styles for `.stars__star` in each `.stars--X .stars__star:nth-child(Y)` selector. Consider creating a base class for `.stars__star` that includes these repeated styles to follow the DRY (Don't Repeat Yourself) principle.